### PR TITLE
Fix TREC-COVID regressions

### PR DESCRIPTION
**Pull request by lintool**
These regressions hadn't been run in a while - since upgrade to Lucene 9. Changed paths and tweaked expected scores.

### CODE REVIEW

**Comment by lintool** [comment]
Similar to #2195 - we're not able to exactly reproduce... but at least the scripts still run...

These are the commands I ran:

```
# https://github.com/castorini/anserini/blob/master/docs/experiments-covid.md

python src/main/python/trec-covid/download_indexes.py --date 2020-07-16 &
python src/main/python/trec-covid/download_indexes.py --date 2020-06-19 &
python src/main/python/trec-covid/download_indexes.py --date 2020-05-19 &
python src/main/python/trec-covid/download_indexes.py --date 2020-05-01 &
python src/main/python/trec-covid/download_indexes.py --date 2020-04-10 &

nohup python src/main/python/trec-covid/generate_round5_baselines.py >& logs/log.trec-covid.round5 &
nohup python src/main/python/trec-covid/generate_round4_baselines.py >& logs/log.trec-covid.round4 &
nohup python src/main/python/trec-covid/generate_round3_baselines.py >& logs/log.trec-covid.round3 &
nohup python src/main/python/trec-covid/generate_round2_baselines.py >& logs/log.trec-covid.round2 &
nohup python src/main/python/trec-covid/generate_round1_baselines.py >& logs/log.trec-covid.round1 &

# https://github.com/castorini/anserini/blob/master/docs/experiments-covid-doc2query.md

python src/main/python/trec-covid/download_doc2query_indexes.py --date 2020-07-16 &
python src/main/python/trec-covid/download_doc2query_indexes.py --date 2020-06-19 &

nohup python src/main/python/trec-covid/generate_round5_doc2query_baselines.py >& logs/log.trec-covid.round5-docTTTTTquery &
nohup python src/main/python/trec-covid/generate_round4_doc2query_baselines.py >& logs/log.trec-covid.round4-docTTTTTquery &
```

File: src/main/python/trec-covid/covid_baseline_tools.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def evaluate_runs(qrels, runs, expected={}, check_md5=True):
                         'judged_cut_20', 'map', 'recall_1000', 'judged_cut_1000']:
                 if key in expected[run]:
                     assert metrics[key] == expected[run][key],\
```
Comment (ronakice) [inline src/main/python/trec-covid/covid_baseline_tools.py:238]: shouldn't this be noted with first four decimal places too?

**Review by ronakice — APPROVED** [review]
LGTM just this question wherever such code exists

File: src/main/python/trec-covid/covid_baseline_tools.py (hunk 1 above)
Comment (lintool) [reply src/main/python/trec-covid/covid_baseline_tools.py]: Addressed your comment with ` math.isclose`

**Pull request merged**